dockers/docker-ptf/Dockerfile.j2

```diff
@@ -139,6 +139,13 @@ RUN ln -s /usr/bin/tcpdump /usr/sbin/tcpdump
 
 RUN mkdir -p /var/log/supervisor
 
+# add dependencies to build gnmi pyclient
```

Comment:
We're not really adding dependencies or building the client, correct? If we're simply installing the GNMI Python client, then the comment should read, `# Install Python-based GNMI client`

Reply:
updated